src/components/Security/Profiles/CommonList.vue

```diff
@@ -42,144 +41,155 @@
+        return false
       }
-    },
-    computed: {
-      displayBulkDelete () {
-        return this.selectedDocuments.length > 0
-      },
-      allChecked () {
-        if (!this.selectedDocuments || !this.documents) {
-          return false
-        }
 
-        return this.selectedDocuments.length === this.documents.length
-      },
-      basicFilter () {
-        try {
-          return JSON.parse(this.$store.state.route.query.basicFilter)
-        } catch (e) {
-          return null
-        }
-      },
-      rawFilter () {
-        try {
-          return JSON.parse(this.$store.state.route.query.rawFilter)
-        } catch (e) {
-          return null
-        }
-      },
-      paginationFrom () {
-        return parseInt(this.$store.state.route.query.from) || 0
-      },
-      paginationSize () {
-        return parseInt(this.$store.state.route.query.size) || 10
+      return this.selectedDocuments.length === this.documents.length
+    },
+    basicFilter() {
+      try {
+        return JSON.parse(this.$store.state.route.query.basicFilter)
+      } catch (e) {
+        return null
+      }
+    },
+    rawFilter() {
+      try {
+        return JSON.parse(this.$store.state.route.query.rawFilter)
+      } catch (e) {
+        return null
       }
     },
-    methods: {
-      isChecked (id) {
-        return this.selectedDocuments.indexOf(id) > -1
-      },
-      toggleAll () {
-        if (this.allChecked) {
-          this.selectedDocuments = []
-          return
-        }
+    paginationFrom() {
+      return parseInt(this.$store.state.route.query.from) || 0
+    },
+    paginationSize() {
+      return parseInt(this.$store.state.route.query.size) || 10
+    }
+  },
+  methods: {
+    isChecked(id) {
+      return this.selectedDocuments.indexOf(id) > -1
+    },
+    toggleAll() {
+      if (this.allChecked) {
         this.selectedDocuments = []
-        this.selectedDocuments = this.documents.map((document) => document.id)
-      },
-      toggleSelectDocuments (id) {
-        let index = this.selectedDocuments.indexOf(id)
+        return
+      }
+      this.selectedDocuments = []
+      this.selectedDocuments = this.documents.map(document => document.id)
+    },
+    toggleSelectDocuments(id) {
+      let index = this.selectedDocuments.indexOf(id)
 
-        if (index === -1) {
-          this.selectedDocuments.push(id)
-          return
-        }
+      if (index === -1) {
+        this.selectedDocuments.push(id)
+        return
+      }
 
-        this.selectedDocuments.splice(index, 1)
-      },
-      hasSearchFilters () {
-        return this.$store.state.route.query.searchTerm !== '' || this.basicFilter.length > 0 || this.rawFilter.length > 0
-      },
-      fetchData () {
-        let pagination = {
-          from: this.paginationFrom,
-          size: this.paginationSize
-        }
+      this.selectedDocuments.splice(index, 1)
+    },
+    hasSearchFilters() {
```

Comment:
should be a computed value

Reply:
It definitely should be, but doesn't seem to be used anywhere. Again, this is not my code, I just formatted it.